deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/config.json

```diff
@@ -85,7 +85,7 @@
     "HttpSyncRetryWaitMax": "30s",
     "HttpSyncRetryWaitMin": "1s",
     "HttpSyncTimeout": "10s",
-    "IngestWorkerCount": 20,
+    "IngestWorkerCount": 10,
```

Comment:
This is a double edge sword because it impacts both fetching advertisements _and_ processing them.

Reducing it means less write parallelism but also potentially slower ad chain download.

We should engineer a tuning knob to offer individual control for each, considering one is disk IO bound and the other is network bound.

Cc @gammazero

Reply:
see https://github.com/ipni/storetheindex/issues/1504#issuecomment-1493220011